File: skills/write-readable-code/assets/layout/js-ts/template.ts
import { dependency } from "./dependency.js"; // stubbed by verify.sh

export const PUBLIC_CONSTANT = 1;
const PRIVATE_CONSTANT = 2;

export class PublicType {
	#state = 0;

	constructor() {}

	publicMethod(): void {}

	#privateMethod(): void {}
}

class PrivateType {
	#helper(): void {}
}

function privateHelper(): void {}

export function publicFunction(): void {}

File: skills/write-readable-code/assets/layout/swift/template.swift
import Foundation

public let publicConstant = 1
internal let moduleConstant = 2
private let fileConstant = 3

public final class PublicType {
    public init() {}

    public func publicMethod() {}

    internal func moduleMethod() {}

    fileprivate func fileMethod() {}

    private func privateMethod() {}
}

private struct PrivateType {
    func helper() {}
}
